apps/pie-microsite/src/content/pages/foundations/elevation/overview.md

```diff
@@ -36,14 +36,18 @@ alt: "Visual representation of the difference in hierarchy achieved by using $el
 caption: "Visual representation of the difference in hierarchy achieved by using $elevation-04 and $elevation-01."
 } %}
 
-### Don’t overdo it
+### Use elevation to help distinguish overlapping elements
 
-Using elevation consistently but sparingly will help your users understand the hierarchy and interaction patterns for the design elements in your product. Using too many elevation tokens in your designs will only empoverish the user’s experience.
+For components which appear on top of other elements of the design, try using elevation to make them more visible.
 
 {% contentPageImage {
 src:"../../../../../assets/img/foundations/elevation/overlapping-elements.svg",
 mobileSrc: "../../../../../assets/img/foundations/elevation/overlapping-elements_narrow.svg",
 width: "850px",
 alt: "Visual representation of an Icon Button floating over a carousel of cards.",
 caption: "Visual representation of an Icon Button floating over a carousel of cards."
-} %}
\ No newline at end of file
+} %}
+
+### Don’t overdo it
+
+Using elevation consistently but sparingly will help your users understand the hierarchy and interaction patterns for the design elements in your product. Using too many elevation tokens in your designs will only empoverish the user’s experience.
```

Comment:
"Empoverish" is a peculiar choice of word for this sentence 😄

Reply:
very eloquent your majesty innit? :P

Reply:
I'd probably go with `diminish` personally, but not sure who we need to check copy changes with

Reply:
Either way, I think it's actually spelled `impoverish`, but I'd still go with `worsen`.

Reply:
Can make this change, will let @raoufswe know and update it